controller/sync_strats/sync_instant.go

```diff
@@ -0,0 +1,528 @@
+/*
```

Comment:
This file represents the sync logic. It is an enhanced version of what we had prior to this that supports

- resync requests
- sequencing sync messages
- enhanced logging
- sets edge router sync states